src/main/java/Bonnie/Storage.java

```diff
@@ -0,0 +1,80 @@
+package Bonnie;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Storage {
+
+    static private File f;
+
+    public Storage() {
+    }
+
+    // Prepare the file for reading and writing
+    protected static void init() {
+        f = new File("src/main/java/Files/tasks.txt");
```

Comment:
Would recommend storing the file path as a variable since you are using it in multiple places